staging/src/k8s.io/kubectl/pkg/cmd/create/create_cronjob.go

```diff
@@ -185,6 +187,10 @@ func (o *CreateCronJobOptions) Run() error {
 		if err != nil {
 			return fmt.Errorf("failed to create cronjob: %v", err)
 		}
+	} else {
```

Comment:
Honestly, I'd do this in `createCronJob` method, that's what I'd love to see happening everywhere.

Reply:
Ok, I'll change it.